pages/data-design/defining-descriptive-events-and-properties.mdx

```diff
@@ -102,12 +100,7 @@ For example for a "Search Result Position" property on a "Search Result Clicked"
 
 ## <a name="property-constraints"></a> Property constraints
```

Comment:
Don’t think we need these anymore, the TOC component just digs the #s out

Reply:
Yeah lets kill'em in a follow-up PR